src/components/ArtifactSigner/artifact-signer.tsx

```diff
@@ -0,0 +1,270 @@
+import './artifact-signer.scss';
+import {
+  Button,
+  ProgressStep,
+  ProgressStepper,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import {
+  ArrowLeftIcon,
+  ExternalLinkSquareAltIcon,
+} from '@patternfly/react-icons';
+import React from 'react';
+
+type GettingStartedProps = {
+  onGetStartedClick: () => void;
+};
+
+type ArtifactSignerProps = {
+  currentStep: number;
+  onDownloadSecret: () => void;
+  onDocumentationClick: () => void;
+  userHasDownloadedSecret: boolean;
+  nextStep: () => void;
+  previousStep: () => void;
+};
+
+const GettingStarted = ({ onGetStartedClick }: GettingStartedProps) => {
+  return (
+    <div>
+      <TextContent>
+        <Text component={TextVariants.p}>
+          {/* Trusted Artifact Signer enables secure signing and verification of software artifacts, enhancing supply chain security. Signed materials are stored in a
+          tamper-resistant public log. */}
+          Trusted Artifact Signer empowers software developers and consumers to
+          securely sign and verify software artifacts such as release files,
+          container images, binaries, bill of material manifests and more to
+          enhance software supply chain security. Signing materials are then
+          stored in a tamper-resistant public log.
```

Comment:
Can some form of this be moved to https://github.com/RedHatInsights/dbaas-ui/pull/52/files#diff-e26c3fd6dd4192c6a8517644bcb299bb20531c6b0da219ff8f644f67fc152f32R35-R37 so that it's visible on every screen?